Assignment2/src/classes/TrainingSessionManager.cls

```diff
@@ -0,0 +1,194 @@
+/** 
+    * Purpose: (Assignment 2) Class To Perform Create, Read, Update, Delete on Training_Session__c.
+    * Created by: Priyank Kumawat
+    * Created Date: 27-11-24
+    * Revision Log: V:1.1
+**/
+public with sharing class TrainingSessionManager {
+    public enum Sessions {WEBINAR , WORKSHOP , SEMINAR}
+    
+    /**
+        * Adds attendees to a training session and use of enum Sessions.
+        * @param  sessionName of the Training_Session__c record.
+        * @param  date of the training sesstion.
+        * @param  location of the training session.
+        * @param  noOfSesstion of the training session.
+        * @throws Error During Insertion:
+    **/
+    public static void createRecordForTrainingSession(String sessionName, Sessions typeOfSession,String dateOfSesstion, String location, Integer noOfSesstion) {
```

Comment:
Try to adjust the method parameters in different line, so it will not override the line and look good.